### PR TITLE
Update vpp-apps-ios.md

### PR DESCRIPTION
**Pull request by alanburchill**
Apple has updated the business portal. The option is now listed on "Content Tokens" not "Server Tokens" on the page. Also added a Note to highlight that the expiration date does not immediately reflect the update and therefore may lead to much confusion that the renew does not work.

### CODE REVIEW

File: memdocs/intune/apps/vpp-apps-ios.md [hunk 1]
```diff
@@ -219,10 +219,14 @@ You can renew an Apple Business Manager location token (Apple VPP token) by down
...
-3. Update the token in [Microsoft Intune admin center](https://go.microsoft.com/fwlink/?linkid=2109431) by selecting **Tenant administration** > **Connectors and tokens** > **Apple VPP tokens**.
-4. Select the VPP token you are renewing, click **Edit** on the Basics category, upload the new token on this page, and then save your changes.
+2. Navigate to **Apple Business (or School) Manager**, by selecting **Preferences** > **Payments and Billing** > **Apps and Books** > **Content  Tokens**.
+3. Click on **Download** and save the token file.
+4. Update the token in [Microsoft Intune admin center](https://go.microsoft.com/fwlink/?linkid=2109431) by selecting **Tenant administration** > **Connectors and tokens** > **Apple VPP tokens**.
+5. Select the VPP token you are renewing, click **Edit** on the Basics category, upload the new token on this page, and then save your changes.
 
+> [!NOTE]
+> The expiration date listed in InTune may take some time to reflect the new certificate expiration date after renew.  To confirm the renew was successful refresh the page until the expiration date has been updated.
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/vpp-apps-ios.md:228]:
```suggestion
> The expiration date listed in InTune may take some time to reflect the new certificate expiration date after renewal.  To confirm the renewal was successful, refresh the page until the expiration date has been updated.
```

File: memdocs/intune/apps/vpp-apps-ios.md [hunk 2]
```diff
@@ -219,10 +219,14 @@ You can renew an Apple Business Manager location token (Apple VPP token) by down
...
+   
 > [!NOTE]
 > You must renew the existing Apple VPP token or location token when the user who set up the token in Apple Business Manager changes their password or the user leaves your Apple Business Manager organization. Tokens that are not renewed will show "invalid" status in Intune.
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/vpp-apps-ios.md:231]:
```suggestion
> - The expiration date listed in InTune may take some time to reflect the new certificate expiration date after renewal.  To confirm the renewal was successful refresh the page until the expiration date has been updated.
> - You must renew the existing Apple VPP token or location token when the user who set up the token in Apple Business Manager changes their password or the user leaves your Apple Business Manager organization. Tokens that are not renewed will show an invalid status in Intune.
```

**Comment by yogkumgit** [comment]
@alanburchill Gentle reminder.

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
#sign-off

**Comment by shdyas** [comment]
#sign-off

**Pull request merged**